db/migrate/20231201091348_add_code_and_name_to_payment_providers.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class AddCodeAndNameToPaymentProviders < ActiveRecord::Migration[7.0]
+  def up
+    add_column :payment_providers, :code, :string
```

Comment:
Rails/BulkChangeTable: You can use `change_table :payment_providers, bulk: true` to combine alter queries. (https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_table, https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html)